ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenter.kt

```diff
@@ -0,0 +1,14 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import androidx.compose.runtime.Composable
+import com.revenuecat.purchases.ui.revenuecatui.ExperimentalPreviewRevenueCatUIPurchasesAPI
+
+/**
+ * Composable offering a full screen Customer Center UI configured from the RevenueCat dashboard.
+ */
+@Composable
+@ExperimentalPreviewRevenueCatUIPurchasesAPI
+// CustomerCenter WIP: Make public when ready
+internal fun CustomerCenter() {
```

Comment:
Yeah, we probably should, however, I was wondering wether we should add it now or once we have actual parameters for it? Right now, that modifier would be mostly unused I think... since we still don't have anything we would like to customize? Not sure if we want to allow changing the size of the internal container or something like that... Just worried that devs can set extra modifiers and they change the UI in unexpected ways...

Reply:
Yea we would pass it down all the way to `CustomerCenterScaffold`, which will pas it to the `Column` (which is the first actual UI that gets emitted). I personally think this is part of being a good citizen of the Compose landscape haha. If devs want to set an alpha value, custom background color, or anything else, who are we to stop them?

It also makes it a lot easier to integrate `CustomerCenterView()`, if only because you'd have full control over the size. What if devs want to show it in a [list-detail](https://m3.material.io/foundations/layout/canonical-layouts/list-detail) layout on a tablet? (The left side of the screen would be the main list of settings containing a button to open the customer center, which then opens on the right side of the screen.) I think it's near-impossible to anticipate all of these scenarios, and adding a `Modifier` makes them all possible.

Reply:
Yeah makes sense, will do this. Thanks again!

Reply:
Nice, thank you! 😄